python-frontend/typeshed_serializer/tests/test_serializers.py

```diff
@@ -64,15 +64,18 @@ def test_importer_serializer():
     assert importer_serializer.is_exception("other", None, set()) is False
 
 
-def test_all_third_parties_are_serialized(typeshed_third_parties):
+def test_third_parties_are_serialized_without_excluded_packages(typeshed_third_parties):
```

Comment:
Here I think it would be nice to assert that `typeshed_third_parties` does not contain anything related to `win32`.